src/components/BreadCrumb/BreadCrumb.tsx

```diff
@@ -0,0 +1,66 @@
+import { Breadcrumb, BreadcrumbItem, PageBreadcrumb } from '@patternfly/react-core';
+import { isUndefined } from 'lodash';
+import { useMatches } from 'react-router-dom';
+
+import { TBreadCrumbAttributes, breadCrumbAttributes } from 'common/breadCrumbAttributes';
+
+import { uiLogger } from 'services/uiLogger';
+
+export interface ISingleBreadCrumb {
+  entity: string;
+  data: string | undefined;
+}
+
+export interface IBreadCrumbProps {
+  pageBreadCrumbs: ISingleBreadCrumb[];
+}
+
+export const BreadCrumb = ({ pageBreadCrumbs }: IBreadCrumbProps) => {
+  const matches = useMatches();
+  const matchCrumbs = matches.filter((match) => !isUndefined(match.handle));
+  const pageCrumbs = pageBreadCrumbs as ISingleBreadCrumb[];
+
+  const breadCrumbs = matchCrumbs.map((crumb, index) => {
+    const notLast = index < matchCrumbs.length - 1;
+    if (crumb.handle && typeof crumb.handle === 'string') {
+      const crumbHandle: TBreadCrumbAttributes = crumb.handle as TBreadCrumbAttributes;
+
+      let text;
+      if (pageCrumbs) {
+        const filteredCrumb = pageCrumbs.filter((pCrumb) => pCrumb.entity === crumbHandle);
+        if (filteredCrumb.length === 0) {
+          text = breadCrumbAttributes[crumbHandle].title;
+        } else if (filteredCrumb.length === 1) {
+          text = filteredCrumb[0].data
+            ? filteredCrumb[0].data
+            : breadCrumbAttributes[filteredCrumb[0].entity as TBreadCrumbAttributes].title;
+        } else {
+          uiLogger.error('More crumbs of same type comming from pages.');
+        }
+      }
+      if (notLast) {
+        return (
+          <BreadcrumbItem key={crumb.id} to={crumb.pathname}>
+            {text}
+          </BreadcrumbItem>
+        );
+      }
+      return (
+        <BreadcrumbItem key={crumb.id} isActive to={crumb.pathname}>
+          {text}
+        </BreadcrumbItem>
+      );
+    } else {
+      uiLogger.log("Breadcrumb useMatch 'handle' is null or not string, this option is not supported.");
+      return [];
+    }
+  });
+  if (!breadCrumbs[1]) {
+    return null;
+  }
```

Comment:
did you mean `breadCrumbs.length === 0` (`!breadCrumbs.length`) ?
or why should there be second element?

Reply:
We're skipping bread crumbs on first level pages like products,projects page etc. and showing them from product detail level of pages. That's the reason for it, to ignore first level.